src/compas/files/amf.py

```diff
@@ -2,30 +2,30 @@
 from __future__ import absolute_import
 from __future__ import division
 
+from compas.files._BaseReader import BaseReader
```

Comment:
After the previously suggested rename:
```suggestion
from compas.files.base_reader import BaseReader
```